MyNumSetLibrary/MyNumSetLibrary.Tests/SetWriterTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using MyNumSetLibrary.Writer;
+using System.IO;
+
+namespace MyNumSetLibrary.Tests
+{
+    [TestClass]
+    public class SetWriterTests
+    {
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void SetWriter_Constructor_ThrowsArgumentNullException()
+        {
+            //Assign
+            _ = new SetWriter(null!);
+
+            //Act
+
+            //Assert
+        }
+
+        [TestMethod]
+
+        public void SetWriter_Constructor_FileExists()
+        {
+            //Assign
+            string filePath = Path.GetTempFileName();
+            _ = new SetWriter(filePath);
+
+            //Act
+
+            //Assert
+            Assert.IsTrue(File.Exists(filePath));
```

Comment:
This is passing, not because of anything in the `SetWriter` constructor but because [Path.GetTempFileName()](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.gettempfilename?view=net-5.0) actually "creates a uniquely named, zero-byte temporary file on disk and returns the full path of that file."